core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -735,6 +735,32 @@ void processMeasurementEvent(WriterJob job) {
                             final int colType = writer.getMetadata().getColumnType(colIndex);
                             if (ColumnType.isString(colType)) {
                                 row.putStr(colIndex, job.floatingCharSink);
+                            } else if (ColumnType.isGeoHash(colType)) {
+                                int typeChars = GeoHashes.getBitsPrecision(colType) / 5;
```

Comment:
can we have this pre-computed? otherwise we divide by 5 needlessly on every value